### PR TITLE
Add Account Disabling Tutorial

### PR DESCRIPTION
**Pull request by YvonneW**
Added  the following files:

- learn/configuring-account-disabling.md
- assets/img/learn/account-disabled-claim-supported-by-default.png
- assets/img/learn/account-disabled-edit.png
- assets/img/learn/account-disabled-email.png
- assets/img/learn/account-disabling-form.png
- assets/img/learn/account-disabling-option.png
- assets/img/learn/account-enabled-email.png
- assets/img/learn/user-disabled.png

Updated mkdocs.yml and managing-user-accounts.md.

### CODE REVIEW

File: en/docs/learn/configuring-account-disabling.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+		<img src="../../assets/img/learn/role-permissions-option.png" alt="Role Permissions option" width="700" style="border:1px solid grey"> 	 		 		
+
+	3.	Select **Login** and click **Update**.
+
+		<img src="../../assets/img/learn/login-permission.png" alt="Login permission" width="700" style="border:1px solid grey">  
+
+3.	To disable Larry's user account:
+	
+	1.	Click **User Profile** option of Larry.
```
Comment (gomathyk) [inline en/docs/learn/configuring-account-disabling.md]: Shall we remove ".. of Larry"?

File: en/docs/learn/configuring-account-disabling.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+		<img src="../../assets/img/learn/user-profile-option.png" alt="User Profile option" width="700" style="border:1px solid grey">   
+
+	2.	Enter an email address to which Larry's account disabling emails will be sent and select the **User Locked** check box.
+
+		<img src="../../assets/img/learn/user-disabled.png" alt="User Disabled option" width="700" style="border:1px solid grey">    		
+
+	3. 	Click **Update**. 
+
+	4.	An email that informs about the account disabling is sent to the given email address.
```
Comment (gomathyk) [inline en/docs/learn/configuring-account-disabling.md]: about the disabled account?

File: en/docs/learn/configuring-account-disabling.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+	6.	Try logging in with Larry's credentials. Note that an error message appears.
+
+4. To revoke the account disabling:
+
+	1.	Click **User Profile** option of Larry.
+
+	2.	Unselect the **User Disabled** check box.
+
+	3.	Click **Update**
```
Comment (gomathyk) [inline en/docs/learn/configuring-account-disabling.md]: fullstop

File: en/docs/learn/configuring-account-disabling.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+	4.	An email that informs about the account disabling is sent to the given email address.
```
Comment (gomathyk) [inline en/docs/learn/configuring-account-disabling.md]: "about the disabled account" would be better IMO. WDYT?

File: en/docs/learn/configuring-account-disabling.md (hunk 1 above)
Comment (YvonneW) [reply en/docs/learn/configuring-account-disabling.md]: This is required as there will be other users that have the User Profile button. (These are user rows)

File: en/docs/learn/configuring-account-disabling.md (hunk 4 above)
Comment (YvonneW) [reply en/docs/learn/configuring-account-disabling.md]: It should have been the enabled account. :D But, thanks for pointing to this line!

**Pull request merged**